File: Exercises/exercism.org/c/resistor-color/resistor_color.c
#include "resistor_color.h"

resistor_band_t array_of_colors[] = { BLACK, BROWN, RED,    ORANGE, YELLOW, GREEN, BLUE,  VIOLET, GREY,   WHITE };

 const resistor_band_t * colors()
 {
    return &array_of_colors[0];
 }

 int color_code(resistor_band_t color)
 {
    return (int)color;
 }

File: Exercises/exercism.org/zig/resistor-color/resistor_color.zig
pub const ColorBand = enum(usize) {
    black = 0,
    brown = 1,
    red = 2,
    orange = 3,
    yellow = 4,
    green = 5,
    blue = 6,
    violet = 7,
    grey = 8,
    white = 9,
};

pub fn colorCode(color: ColorBand) usize {
    return @enumToInt(color);
}

pub fn colors() []const ColorBand {
    var result = [_]ColorBand{
        .black, .brown, .red,    .orange, .yellow,
        .green, .blue,  .violet, .grey,   .white,
    };
    return result;
}